teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote>Determine <s>the querying</s> <strong>a query </strong>compound&nbsp;</blockquote><blockquote>we will search<s> the similar</s> compounds <s>of</s> <strong>similar to </strong>Gefitinib</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>done</p>